app/build.gradle

```diff
@@ -218,10 +218,6 @@ dependencies {
 kotlinter {
     ignoreFailures = false
     reporters = ['checkstyle', 'plain']
-    experimentalRules = true
-    disabledRules = [
-            "no-wildcard-imports"
```

Comment:
These configuration options are not supported in build.gradle any more; they are now read from .editorconfig (see above).